src/App.jsx

```diff
@@ -1,7 +1,36 @@
+import { useState, useEffect } from 'react';
+import jsonData from './data.json';
+import './App.css';
+
 const App = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    setData(jsonData);
+  }, []);
+
   return (
     <>
-      <h1>Vite + React</h1>
+      <div className="wrapper">
+        {data.map((item) => (
+          <>
+            <h1 className="title" key={item.id}>
+              <span>{item.id}.</span>
+              {item.question}
+            </h1>
+            <div className="answers">
+              {item.answers.map((answer) => {
+                return (
+                  <button className="button" key={answer.id}>
```

Comment:
Make Button separate component 
Also each item should be in separated component 
data.map(item => <Component />)